openmetadata-service/src/main/java/org/openmetadata/service/apps/NativeApplication.java

```diff
@@ -0,0 +1,16 @@
+package org.openmetadata.service.apps;
+
+import org.openmetadata.schema.entity.app.Application;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+
+public interface NativeApplication extends Job {
+  void init(Application app, CollectionDAO dao);
+
+  void triggerOnDemand(Object requestObj);
```

Comment:
what is the requstObj here? why are we not passing a specific class instead of an Object

Reply:
removed , on trigger I was taking in dynamic config as per app config, but I think for now we can always read from installation of the app, and the user can update there itself if they need new config